core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -229,6 +231,9 @@ template <typename DriverType, unsigned int MaxThreadsPerBlock,
 struct HIPParallelLaunchKernelFunc<
     DriverType, Kokkos::LaunchBounds<MaxThreadsPerBlock, MinBlocksPerSM>,
     HIPLaunchMechanism::LocalMemory> {
+  using base_t = HIPParallelLaunchKernelFuncData<
```

Comment:
`base_t` is misleading, it is not the base class that `HIPParallelLaunchKernelFunc` derives from.